CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@
 
 - **User Story**: As a developer, I want to be able to use the Solid Design System without having to ask for help.
 - **Actions**:
-  - Provide stories for props, slots etc. in Storybook. [Explore components and interact with them](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/).
+  - Provide stories for props, slots etc. in Storybook
```

Comment:
would it make sense (as the point has been mentioned in #622) to explain a bit more detailed what we are using the stories for?